doc/man3/EVP_SIGNATURE.pod

```diff
@@ -34,6 +35,7 @@ EVP_SIGNATURE_gettable_ctx_params, EVP_SIGNATURE_settable_ctx_params
  const char *EVP_SIGNATURE_get0_description(const EVP_SIGNATURE *signature);
  const OSSL_PARAM *EVP_SIGNATURE_gettable_ctx_params(const EVP_SIGNATURE *sig);
  const OSSL_PARAM *EVP_SIGNATURE_settable_ctx_params(const EVP_SIGNATURE *sig);
+ int EVP_SIGNATURE_has_message_update(const EVP_SIGNATURE *sig);
```

Comment:
The function also needs to be mentioned in the `HISTORY` section.

Reply:
Done.